resultsdb_frontend/static/js/search.js

```diff
@@ -70,38 +70,23 @@ $('document').ready(function() {
             $('#outcome option[value="'+item+'"]').attr('selected','selected');
         });
     }
-    if(qs.since){
-        dates = qs.since.split(',');
-        start = moment(dates[0], 'YYYY-MM-DD');
-        end = moment();
-        if (dates.length > 1){
-            end = moment(dates[1], 'YYYY-MM-DD');
-        }
-    }
-    else {
-        start = moment().subtract(29, 'days');
-        end = moment();
-    }
-    since = start.format('YYYY-MM-DD') + 'T00:00:00,' + end.format('YYYY-MM-DD') + 'T23:59:59';
 
-    $("#searchrange").daterangepicker({
-        startDate: start,
-        endDate: end,
-        ranges: {
-           'Today': [moment(), moment()],
-           'Last 7 Days': [moment().subtract(6, 'days'), moment()],
-           'Last 30 Days': [moment().subtract(29, 'days'), moment()],
-        }
-    }, function(start, end, label) {
-        if (label === 'Custom Range'){
-            //$('#searchrange').val(start.format('MMMM D, YYYY') + ' - ' + end.format('MMMM D, YYYY'));
-            since = start.format('YYYY-MM-DD') + 'T00:00:00,' + end.format('YYYY-MM-DD') + 'T23:59:59';
-        }
-        else{
-            $('#searchrange').val(label);
-            since = start.format('YYYY-MM-DD') + 'T00:00:00,' + end.format('YYYY-MM-DD') + 'T23:59:59';
+    var date_range = function(since) {
```

Comment:
Using `var` is obsolete in JavaScript. You should use `let` or `const` instead. Also, variable names should follow `lowerCamelCase` style, such as `dateRange`, which is recommended for naming.

Reply:
I would prefer to keep the code consistent. Do you have a link to a document which says `var` is obsolete?

Reply:
I've changed this to simpler `function date_range(since)`.

Reply:
See https://google.github.io/styleguide/jsguide.html#naming about naming and https://google.github.io/styleguide/jsguide.html#features-local-variable-declarations about `var` vs `let` and `const` (if you want to know more why it's discouraged to used `var`: https://plainenglish.io/blog/4-reasons-why-var-is-considered-obsolete-in-modern-javascript (TL;DR let is safe in the sense of scoping, (re)declaration, checking whether it was defined or not, while var is not safe))

Reply:
Feel free to create an issue to refactor the js file and perhaps add some checks. Though I currently see it as a minor priority.